src/tutorials/0007-verifying-storage-on-filecoin/03.md

```diff
@@ -0,0 +1,20 @@
+---
+    title: "Proof-of-Replication (PoRep)"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+In a Proof of Replication, a storage miner proves that they are storing a _physically_ unique copy, or _replica_, of the data in a sector, the fundamental storage unit used by Filecoin.
+
+## The pre-commit phase: Replication & CommD
+
+In the _pre-commit_ phase, the storage miner places the piece data received into a sector. Once a sector is full, a CommD (Commitment of Data, aka UnsealedSectorCID) is produced, representing the root node of all the piece CIDs contained in the sector. Most often, this will be an aggregate of pieces from a number of different clients.
+
+## The commit phase: Sealing & CommR
+
+In the _commit_ phase, a process called _sealing_ takes place. Sealing is a sequential operation which gradually generates an encoding of the data for each miner, also known as a unique replica. (Note that encoding is not the same as encryption. If you want to store private data, you must encrypt it before adding it to the Filecoin network.)
```

Comment:
Sealing actually takes place during pre-commit, at least in the way we are using commit and pre-commit in the code.

Reply:
the actual commit phase is getting a challenge from the chain after posting the CommR and responding with the snark proof.

Reply:
Thanks for the feedback! 

@mishmosh How would you like to handle this suggestion? Most people and docs I reference all have different opinions of what happens in what phase. Do you want me to redefine this based on @dignifiedquire's feedback, which might require moving the section on SNARK and adding info on the challenge (which we haven't discussed at all)? Or remove the labels `pre-commit` and `commit` since I'm not getting agreement on what they encompass?

Reply:
I might eliminate the labels altogether, both because of lack of agreement and because they introduce a parallel vocabulary that adds extra complexity.

Reply:
Okay, can do. I don't think we've talked about the challenge yet that @dignifiedquire mentions. What should I add about that?

Reply:
That's probably more detail than is needed in this lesson, but @dignifiedquire feel free to object :)

Reply:
Mosh and I have made  a number of modifications to Lesson 3 to remove the references to pre-commit and commit phases and add more detail about the sealing process. See https://github.com/ProtoSchool/protoschool.github.io/pull/447/commits/f8b2ee5ef5f6e2bf3bd522145058f41086d9a7a5